labgrid/driver/power/phoenix_fl_switch.py

```diff
@@ -0,0 +1,76 @@
+'''
+This Driver was tested on a FL SWITCH 2303-8SP1 with FW-version 3.27.01 BETA
+file   phoenix_fl_switch.py
+author Raffael Krakau
+date   2023-08-24
+
+Copyright 2023 JUMO GmbH & Co. KG
+'''
+import pexpect
+
+PORT = 23
+
+
+def login_telnet(tn):
+    """
+    Login user with set credentials
+
+    @param tn : pyexpect-telnet-object
+    @returns : logged in pyexpect-telnet-object
+    """
+    username = "admin"
+    password = "private"
+
+    # login user with password
+    tn.expect(b'User: ')
+    tn.send(bytes(f'{username}\r\n', "utf-8"))
+    tn.expect(b'Password: ')
+    tn.send(bytes(f'{password}\r\n', "utf-8"))
+    return tn
```

Comment:
returning the same object is not necessary, phyton use call by ref. That means you have changed the origin object

Reply:
thank you for advice/explanation